### ISSUE TITLE
bug(android): can't run the Android Client in virtual device after #6362

### ISSUE THREAD

**Issue by ReactorScram** [CloudFire-LLC/cloudfire#6379]
#### Describe the bug

The app starts up, but once it connects to the portal it goes into a crash loop.

#### To Reproduce

Try to run the tip of `main` in Android Studio.

#### Expected behavior

Expect the Resources list to appear on screen.

#### Screenshots / Logs

In `logcat` it says something about `can_be_disabled` being required but not present. Like it's not deserializing some message from the portal correctly, but I can't understand why.

#### Platform (please complete the following information)

- _Component (i.e. macOS client / Linux client / Gateway / Admin portal):_ Android Client
- _Firezone Version (e.g. 1.0.0 or N/A):_ `main`. `git bisect` is saying #6362 introduced this
- _OS and version: (e.g. Ubuntu 22.04 or N/A):_
- _Deployment method: (e.g. Docker / Systemd / App Store or N/A):_ Local build in Android Studio

#### Additional context

I tried these steps and it still isn't working:
- "Wipe data" of virtual device
- `cargo clean`
- Gradle clean (launched from Android Studio menu)

**Comment by ReactorScram** [comment]
Possible lead

```
Users-MacBook-Pro:firezone user$ git grep can_disable
kotlin/android/app/src/main/java/dev/firezone/android/tunnel/model/Resource.kt:    
    @Json(name = "can_disable") val canBeDisabled: Boolean,
```

**Comment by jamilbk** [comment]
Fixed in #6380
